obspy/io/gcf/src/gcf_io.c

```diff
@@ -16,7 +16,7 @@
 //    2017-05-08   Removed BUG in new merging algorithm in add_GcfSeg that triggered when
 //                  blocks in gcf file are not ordered in time (they usually are), cps
 //    2017-05-23   Removed BUG in new merging algorithm in add_GcfSeg that triggered when
-//                  data vectors had not been decoded --- Note, this diables new scheme
+//                  data vectors had not been decoded --- Note, this disables new scheme
```

Comment:
this is a vendored file not written by us and thus should not be changed for typos IMHO